code/_globalvars/traumas.dm

```diff
@@ -126,10 +126,9 @@ GLOBAL_LIST_INIT(phobia_objs, list(
 		/obj/item/clothing/under/plasmaman/clown
 	)),
 
-	"greytide" = typecacheof(list(/obj/item/clothing/under/color/grey, /obj/item/melee/baton/security/cattleprod,
-		/obj/item/spear, /obj/item/clothing/mask/gas, /obj/item/toy/figure/assistant,
-		/obj/structure/statue/sandstone/assistant
-	)),
+	"greytide" = (typecacheof(list(/obj/item/clothing/under/color/grey, /obj/item/melee/baton/security/cattleprod,
+		/obj/item/spear, /obj/item/toy/figure/assistant,
+		/obj/structure/statue/sandstone/assistant))+typecacheof(list(/obj/item/clothing/mask/gas), FALSE, TRUE)),
```

Comment:
This is unreadable, what does this do? What does `FALSE, TRUE` mean? Can you annotate them with named vars? 

Phobia code is so ugly to read, nothing split out.

Reply:
a6f2987 should resolve this?